src/Datadog.Trace.ClrProfiler.Managed/Integrations/AspNet/AspNetMvcIntegration.cs

```diff
@@ -51,143 +54,148 @@ public static Scope CreateScope(ControllerContextStruct controllerContext)
 
             try
             {
-                if (!Tracer.Instance.Settings.IsIntegrationEnabled(IntegrationId))
-                {
-                    // integration disabled, don't create a scope, skip this trace
-                    return null;
-                }
-
                 var httpContext = controllerContext.HttpContext;
 
                 if (httpContext == null)
                 {
                     return null;
                 }
 
-                var newResourceNamesEnabled = Tracer.Instance.Settings.RouteTemplateResourceNamesEnabled;
-                string host = httpContext.Request.Headers.Get("Host");
-                string httpMethod = httpContext.Request.HttpMethod.ToUpperInvariant();
-                string url = httpContext.Request.RawUrl.ToLowerInvariant();
-                string resourceName = null;
-
-                RouteData routeData = controllerContext.RouteData;
-                Route route = routeData?.Route as Route;
-                RouteValueDictionary routeValues = routeData?.Values;
-                bool wasAttributeRouted = false;
-
-                if (route == null && routeData?.Route.GetType().FullName == RouteCollectionRouteTypeName)
+                Span span = null;
+                var security = Security.Instance;
+                // integration disabled, don't create a scope, skip this trace
```

Comment:
This comment needs to be reversed now :)

Reply:
Done in 5afc27f2, I think you'll enjoy the new message.